projects/storyteller/webapp/src/app/app.component.scss

```diff
@@ -1,3 +1,4 @@
 .container {
-    background-color: #eee;
+  background-color: #eee;
 }
+
```

Comment:
I think two empty lines it's too much :)

Reply:
Done

Reply:
Could you also find all other places with two empty lines?